pcap-linux.c

```diff
@@ -3530,16 +3462,12 @@ activate_sock(pcap_t *handle, int is_any_device)
 	return status;
 }
 
-#ifdef HAVE_PACKET_RING
 /*
- * Attempt to activate with memory-mapped access.
+ * Activate with memory-mapped access.
```

Comment:
The attempt may fail, which is why I said "Attempt to activate".

Reply:
Noted. Sorry.
I was planning to rebase to fix any issues, how should I handle your commit?
Should I keep the mistake in the original commit or just squash the fix with it?

Also, I had no idea you could commit to the PR, I thought that came only from my fork.

Reply:
Whichever is more convenient; if they're equally committed, squash.